.github/ISSUE_TEMPLATE/bump_gmt_checklist.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bump GMT version checklist
+about: Checklist for bumping the minimum required GMT version.
+title: Bump to GMT X.Y.Z
+labels: maintenance
+assignees: ''
+
+---
+
+:tada: [GMT X.Y.Z](https://github.com/GenericMappingTools/gmt/releases/tag/X.Y.Z) has been released! It is installable from the
+[conda-forge channel](https://anaconda.org/conda-forge/gmt/files) using the following command:
+
+`conda install -c conda-forge gmt=X.Y.Z`
+
+<!-- Please add specific checklist items for the tests, xfail pytest markers, and deprecated syntax that need to be updated. -->
+
+**To-Do**:
+- [ ] Bump the minimum required GMT version (1 PR)
+  - [ ] Update `.github/workflows/cache_data.yaml`
+  - [ ] Update `.github/workflows/ci_docs.yml`
+  - [ ] Update `.github/workflows/ci_tests.yaml`
+  - [ ] Update `doc/install.rst`
+  - [ ] Update `environment.yml`
+  - [ ] Update `required_version` in `pygmt/clib/session.py`
+  - [ ] Update `test_get_default` in `pygmt/tests/test_clib.py`
+  - [ ] Update compatibility table in `README.rst`
+- [ ] Fix failing tests (1 or more PRs)
```

Comment:
'Fix failing tests' is listed after 'Bump the minimum GMT version'. Does this imply that we are ok with the CI tests breaking on the main branch for a period of time?

Reply:
I think the only way the tests wouldn't fail for a period of time is if all the changes were in a single PR or xfails were added just to be removed shortly after. I would prefer to have the tests break for a short bit in order to split up the tasks and reduce efforts, but of course other opinions are welcome.

Reply:
Guess it depends on the situation (i.e. how many tests are breaking, and how big of a fix is required). Having failing tests on the main branch means that PRs will need to be merged by those with admin permissions (currently 5, maybe 3 active), so we definitely want to keep that period as short as possible.

Reply:
Good point. I agree that we'll want to make the tests pass again as quickly as possible. We can also adjust the issue checklist from what is in the template on a case-by-case basis.